contracts/bridge/contracts/binance/LightClientBridge.sol

```diff
@@ -393,14 +401,27 @@ contract LightClientBridge is Bitfield, ValidatorRegistry, GuardRegistry {
         emit FinalVerificationSuccessful(msg.sender, id);
     }
 
+    /**
+     * @notice Relyer could claim the expired commitment's collateral
+     *  1/10 of `MIN_SUPPORT` will be slashed
+     * @param id the identifier generated by submit commitment
+     */
+    function cleanExpiredCommitment( uint256 id) public {
+        ValidationData storage data = validationData[id];
+        require(block.number > data.blockNumber + 256, "Bridge: Only expired");
+        data.senderAddress.transfer(MIN_SUPPORT * 9 / 10);
```

Comment:
What will happen to the slashed 1/10 ether leaving here, will they be locked in this contract for ever>

Reply:
Yes, Maybe need a vault to store the asset?

Reply:
Use a `SLASH_VAULT` to store the slashed asset.

Reply:
Cause guard only do sigs the commitment and upgrade itselt. Withdraw，Swap or do other tx need another multiSigWallet.